documentation/book/appendix_metrics.adoc

```diff
@@ -37,66 +37,160 @@ endif::Kubernetes[]
 
 === Prometheus
 
-The provided Prometheus https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/prometheus.yaml[`prometheus.yaml`] YAML file describes all the resources required by Prometheus in order to effectively monitor a Strimzi Kafka & ZooKeeper cluster.
-These resources lack important production configuration to run a healthy and highly available Prometheus server.
-They should only be used to demonstrate this Grafana dashboard example.
+link:https://prometheus.io/[Prometheus^] is an open-source set of components for systems monitoring and alerting.
+{ProductName} uses the link:https://github.com/coreos/prometheus-operator[CoreOS Prometheus Operator^] to run Prometheus on Kubernetes.
+This Operator enables you to run a highly available Prometheus server that is suitable for use in production environments.
 
-The following resources are defined:
+==== Deploying the Prometheus Operator on {OpenShiftName}
+
+To deploy the Prometheus Operator in your Kafka cluster, apply the `.yaml` files from the link:https://github.com/coreos/prometheus-operator/tree/master/example/rbac/prometheus-operator[Prometheus CoreOS repository^].
+
+To use a different namespace than that specified in the repository files, use the following commands to download and edit the files from the repository:
+
+[source,shell,subs="+quotes"]  
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-deployment.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-deployment.yaml
```

Comment:
I think that on other places we have Linux and MacOS versions. Should we have them here as well?